jobs/cloud_controller_ng/spec

```diff
@@ -253,6 +253,18 @@ consumes:
   optional: true
 
 properties:
+  export_dyno:
+    description: "Flag. When true export job name and index to DYNO environment variable"
+    default: true
+
+  use_newrelic:
+    description: "Flag. When true use the Nginx newrelic plugin"
+    default: true
+
+  use_nginx_maintenance:
+    description: "Flag. When true run Nginx maintenance jobs"
+    default: true
+
```

Comment:
Hi!  I think it would make more sense to put these properties at the bottom - they seem less important than things like system and app domain, which we would prefer to have near the top